lib/kits/core-ui/components/common/shopping-list.ts

```diff
@@ -221,8 +221,13 @@ const ShoppingList = ({template}) => {
         container
             ?.querySelector('.sgn-shopping-print-list-btn')
             ?.addEventListener('click', () => {
+                document.body.classList.add('sgn-body-print');
                 window.print();
             });
+
+        window.addEventListener('afterprint', () => {
+            document.body.classList.remove('sgn-body-print');
+        });
```

Comment:
It's probably negligible effect in reality, but this could end up adding a listener repeatedly(which won't be cleaned up since it's on the window), right?